src/controls.plugin.js

```diff
@@ -0,0 +1,60 @@
+function wikiaJWPlayerRecommendedControlsPlugin(player, config, div) {
+	var parser = new DOMParser();
+
+	this.player = player;
+	this.container = div;
+	this.config = config;
+	this.muteIcon = parser.parseFromString(wikiaJWPlayerIcons.volumeOff, 'image/svg+xml').documentElement;
+	this.unmuteIcon = parser.parseFromString(wikiaJWPlayerIcons.volumeOn, 'image/svg+xml').documentElement;
+	this.playIcon = parser.parseFromString(wikiaJWPlayerIcons.play, 'image/svg+xml').documentElement;
+	this.pauseIcon = parser.parseFromString(wikiaJWPlayerIcons.pause, 'image/svg+xml').documentElement;
+	this.wikiaControlsElement = document.createElement('div');
+	this.wikiaControlsElement.appendChild(this.muteIcon);
+	this.wikiaControlsElement.appendChild(this.pauseIcon);
+
+	this.unmuteHandler = this.unmuteHandler.bind(this);
+	this.playHandler = this.playHandler.bind(this);
+	this.pauseHandler = this.pauseHandler.bind(this);
+	this.onreadyHandler = this.onreadyHandler.bind(this);
+
+	this.muteIcon.addEventListener('click', this.unmuteHandler);
+	this.pauseIcon.addEventListener('click', this.pauseHandler);
+	this.playIcon.addEventListener('click', this.playHandler);
+	this.unmuteIcon.addEventListener('click', this.unmuteHandler);
+
+	this.player.on('ready', this.onreadyHandler);
+}
+
+wikiaJWPlayerRecommendedControlsPlugin.prototype.onreadyHandler = function () {
+	var isSmallScreen = this.player.getWidth() <= 200;
+	var playerDiv = document.querySelector('.jwplayer');
+
+	if (this.config.showSmallPlayerControls && isSmallScreen) {
```

Comment:
how about moving check if `this.config.showSmallPlayerControls` is `true` to index.js where plugin is registered? I would just leave `isSmallScreen` check here.